src/node_api.cc

```diff
@@ -1714,6 +1733,39 @@ napi_status napi_get_value_string_length(napi_env env,
   return GET_RETURN_STATUS(env);
 }
 
+// Copies a JavaScript string into a LATIN-1 string buffer. The result is the
+// number of bytes copied into buf, including the null terminator. If bufsize
```

Comment:
Now the null terminator is _not_ counted in the `result`. I understand that is done for consistency with the usage that passes a null buffer just to get the string length. But then the comments need to be updated on all 3 functions.